filters/scheduler/lifo_test.go

```diff
@@ -290,3 +300,88 @@ func TestNewLIFO(t *testing.T) {
 		})
 	}
 }
+
+func TestLifoErrors(t *testing.T) {
+	backend := httptest.NewServer(http.HandlerFunc(func(_ http.ResponseWriter, _ *http.Request) {
+		time.Sleep(time.Second)
+	}))
+	defer backend.Close()
+
+	doc := fmt.Sprintf(`aroute: * -> lifo(5, 7, "100ms") -> "%s"`, backend.URL)
```

Comment:
let's see if the timeout 100ms is not too short and result in flaky test

Reply:
I hope it should be fine - backend latency is 10x == 1s